### PR TITLE
First version of application specification

### PR DESCRIPTION
**Pull request by siredmar**

### CODE REVIEW

File: dev/manifests/applications/edge-app-network/edge-app-network-no-runtimes.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        name: simplenginx
+      traits:
+        - type: labels
+          properties:
+            "app": "edge-worker-1"
+    - name: network
+      type: application-network
+      properties:
+        usernames:
+          - edge-worker-1
```
Comment (KairaMiu) [inline dev/manifests/applications/edge-app-network/edge-app-network-no-runtimes.yaml:20]: this is used nowhere? is this intended?

**Review by KairaMiu — COMMENTED** [review]
Perhaps add a small readme file in `dev/manifests/applicaitons` that describes each sample app manifest in just one sentence so that you don't have to go through each one carefully to understand what is being demonstrated. Unfortunately, the names of the files are not quite sufficient.

Alternatively, the main readme can also be expanded (or compared with the current state, filled with something :P)

**Comment by KairaMiu** [comment]
Please add readme instructions how to generate kube vela output.

File: dev/manifests/applications/edge-app-network/edge-app-network-no-runtimes.yaml (hunk 1 above)
Comment (siredmar) [reply dev/manifests/applications/edge-app-network/edge-app-network-no-runtimes.yaml]: This is indeed intended. The username `edge-worker-1` is the name of the application that needs to be allowed to communicate with edgefarm.network.

**Comment by siredmar** [comment]
Added information in the readme about examples and how to develop this.

**Review by KairaMiu — APPROVED** [review]

**Pull request merged**